### ISSUE TITLE
Cloud Worlds - Pause not working correctly

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#18335]
From a customer:

Just now, I tried some testing with the unpauseatfirstlogin. I went into the world webpage, and found that there was an 'unpause' button, suggesting the world was paused. I went in-game, and found the world was notpaused, despite what the button suggested. I went back to the web panel, pushed unpause. The web page told me the unpause was successful, and the button changed to read 'pause'. I very carefully unchecked the unpauseatfirstlogin box, and saved the changes. I reloaded the webpage, and verified that the unpauseatfirstlogin box was still unchecked. I went in world, then alt-tabbed to the world webpage, and clicked the pause button. I alt-tabbed back to in-world, and found that the game was paused. In the eco client, I clicked the disconnect button to return to the title menu. I then went back in world - and the game was unpaused. So it appears that my return to the world caused the world to unpause, even though the unpauseatfirstloging box was unchecked.

At this moment, things seem somehow unsynchronized. When I go in-game, the world is working (not paused). When I go to the world webpage, the page still has the 'unpause' button, as if the world were paused.
The unpauseatfirstlogin box remains unchecked. Which is nice.

tl;dr: I am unable to use the pause button on the world webpage to cause a long-term pause, no matter what the unpauseatfirstlogin box says. After pausing the world, it appears to unpause on the next login. This is a minor challenge. I would like to be able to pause the world during in-class strategy discussions, to force students to focus on the discussion. At this time, I don't seem able to do that.

**Comment by j3fffff** [comment]
pause working correctly in our tests.
but can unsync easily. that will be fixed with this task: https://github.com/StrangeLoopGames/EcoIssues/issues/18388
